### PR TITLE
Add 'dactyl_tracer' keyboard

### PR DESCRIPTION
**Pull request by dunk2k**
## Description

`readme.md` mentions #20517

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* n/a

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/dactyl_tracer/readme.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
+* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
+* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
+
+## QMK Layouts
+
+The following [layout](/layouts/) is supported:
```
Comment (drashna) [inline keyboards/handwired/dactyl_tracer/readme.md:31]: Not sure this should be a link, though. 
```suggestion
The following [community layout](/layouts/) is supported:
```

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/dactyl_tracer/info.json [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    "usb": {
+        "vid": "0xFEED",
+        "pid": "0x1307",
+        "device_version": "0.0.1"
+    },
+    "matrix_pins": {
+        "cols": ["C6", "D7", "E6", "B4", "B5"],
+        "rows": ["F7", "B1", "B3", "B2", "B6"]
+    },
+    "diode_direction": "COL2ROW",
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_tracer/info.json:15]:
```suggestion
    "diode_direction": "COL2ROW",
    "debounce": 15,
```

File: keyboards/handwired/dactyl_tracer/config.h [hunk 3]
```diff
@@ -0,0 +1,13 @@
+// Copyright 2022 mjohns
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 15
+
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_tracer/config.h:6]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/handwired/dactyl_tracer/keymaps/default/config.h [hunk 4]
```diff
@@ -0,0 +1,13 @@
+// Copyright 2022 mjohns
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+/* mouse config */
```
Comment (drashna) [inline keyboards/handwired/dactyl_tracer/keymaps/default/config.h:4]:
```suggestion
#pragma once

/* mouse config */
```

File: keyboards/handwired/dactyl_tracer/keymaps/via/config.h [hunk 5]
```diff
@@ -0,0 +1,13 @@
+// Copyright 2022 mjohns
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+/* mouse config */
```
Comment (drashna) [inline keyboards/handwired/dactyl_tracer/keymaps/via/config.h:4]:
```suggestion
#pragma once

/* mouse config */
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**